### PR TITLE
added login and signup pages

### PR DESCRIPTION
**Pull request by iHarryD**
brewed-store_login-page
brewed-store_signup-page

### CODE REVIEW

File: Pages/Signup/CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  width: 100%;
+}
+
+.--input-animation {
+  position: relative;
+}
+
+.--input-animation .label {
+  background: var(--SECONDARY-BACKGROUND-CLR);
+  color: rgb(158, 158, 158);
```
Comment (dev-ashishsethi) [inline Pages/Signup/CSS/style.css:42]: could have used variable for colors

File: Pages/Signup/CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  padding: 0 0.3rem;
+  position: absolute;
+  left: 1.3rem;
+  bottom: 0.55rem;
```
Comment (dev-ashishsethi) [inline Pages/Signup/CSS/style.css:46]: use pixels for lengths less than 1rem

File: Pages/Signup/CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/* lOGIN-SIGNUP-OUTER-CONTAINER */
+
+.login-signup-outer-container {
+  box-shadow: 5px 5px 5px 0 #00000070;
```
Comment (dev-ashishsethi) [inline Pages/Signup/CSS/style.css:4]: could have used rbga instead of hex color

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

**Review by Debarshi95 — APPROVED** [review]
Looks good. Just checkout the things the other guy said.

File: Pages/Signup/CSS/style.css (hunk 3 above)
Comment (iHarryD) [reply Pages/Signup/CSS/style.css:4]: done

Comment (iHarryD) [reply Pages/Signup/CSS/style.css:4]: I thought hex are loaded faster than other color formats including rgba. Is it not the case?

File: Pages/Signup/CSS/style.css (hunk 1 above)
Comment (iHarryD) [reply Pages/Signup/CSS/style.css:42]: done

**Pull request merged**